assets/js/common.js

```diff
@@ -506,6 +506,14 @@ var hivepress = {
 							dayElem.innerHTML += '<span class="flatpickr-day-label">' + range.label + '</span>';
 							dayElem.className += ' flatpickr-status';
 
+							if (range.hasOwnProperty('class')) {
+								dayElem.className += ' ' + range.class;
+
+								if (range.class === 'last-day' && !dayElem.classList.contains('selected')) {
+									dayElem.className += ' flatpickr-disabled';
```

Comment:
Maybe let's pass the flatpickr-disabled class directly? Because at least naming is booking-specific, while this is core - please check the meaning of flatpickr-disabled class, maybe let's hard-code the logic to prevent selecting a range over 1+ of flatpickr-disabled days without an extra last-day class.

Reply:
No, we can't pass this class directly, because it's using in the logic below